process/track/blockNotarizer_test.go

```diff
@@ -62,9 +86,69 @@ func TestAddNotarizedHeader_ShouldWork(t *testing.T) {
 	assert.Equal(t, &hdr2, bn.GetNotarizedHeaderWithIndex(0, 1))
 }
 
+func TestCleanupWhenMaxCapacityIsReached_ShouldNotCleanNotarizedHeadersWhenMaxCapacityIsNotReached(t *testing.T) {
+	t.Parallel()
+	bn, _ := track.NewBlockNotarizer(&mock.HasherMock{}, &mock.MarshalizerMock{}, mock.NewMultipleShardsCoordinatorMock())
+
+	maxNumHeadersToKeepPerShard := 1200
+
+	for i := uint64(0); i < uint64(maxNumHeadersToKeepPerShard); i++ {
+		bn.AppendNotarizedHeader(&block.Header{Nonce: i})
+	}
+
+	bn.CleanupWhenMaxCapacityIsReached(0)
+	require.Equal(t, maxNumHeadersToKeepPerShard, len(bn.GetNotarizedHeaders()[0]))
+}
+
+func TestCleanupWhenMaxCapacityIsReached_ShouldCleanOldestNotarizedHeaders(t *testing.T) {
+	t.Parallel()
```

Comment:
add empty line after `t.Parallel()` in each test in this file

Reply:
Done